austrakka/components/iam/role/funcs.py

```diff
@@ -0,0 +1,118 @@
+from austrakka.components.iam.shared_funcs import _get_default_tenant, _get_role_by_name
+from austrakka.utils.api import api_get, api_post, api_patch
+from austrakka.utils.enums.privilege_level import (
+    AUSTRAKKA_ADMIN_LEVEL,
+    FUNCTIONAL_ADMIN_LEVEL,
+    USER_LEVEL)
+
+from austrakka.utils.misc import logger_wraps
+from austrakka.utils.output import print_dict
+
+
+@logger_wraps()
+def get_roles(out_format: str):
```

Comment:
Will this need a record Id in the future if people wanna scope or only have scope on project roles for example. There might be another solution to this I might be out of the loop.

Reply:
Future use case is possible but will be handled when needed.